wallet/src/ui/app/redux/slices/permissions/index.ts

```diff
@@ -39,6 +40,29 @@ export const respondToPermissionRequest = createAsyncThunk<
     }
 );
 
+// Todo: move this to the wallet adapter
+// Get all permissions from from storage
+// remove the permission for a given origin
+// set the new permissions in storage
+export const revokeAppPermissionByOrigin = createAsyncThunk<
```

Comment:
@pchrysochoidis is doing something similar in his PR for the current connected dapp. We should make sure that these approaches are compatible.